File: app/src/main/java/com/example/roomdatabase/fragments/HomeFragment.java
package com.example.roomdatabase.fragments;


import android.os.Bundle;
import android.widget.Button;
import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.example.roomdatabase.MainActivity;
import com.example.roomdatabase.R;

public class HomeFragment extends Fragment implements View.OnClickListener {

    private Button BnAddUser;

    public HomeFragment() {

    }


    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        return inflater.inflate(R.layout.fragment_home, container, false);
    }

    @Override
    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
        super.onViewCreated(view, savedInstanceState);

        BnAddUser = view.findViewById(R.id.bn_add_users);
        BnAddUser.setOnClickListener(this);

    }

    @Override
    public void onClick(View view) {

        switch (view.getId()){

            case R.id.bn_add_users:
            MainActivity.fragmentManager.beginTransaction().replace(R.id.fragment_container,new AddUserFragment()).commit();
            break;
        }
    }
}

File: app/src/main/java/com/example/roomdatabase/fragments/HomeFragment.kt
package com.example.roomdatabase.fragments


import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import com.example.roomdatabase.R
import kotlinx.android.synthetic.main.fragment_home.*


class HomeFragment : Fragment() {

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        return inflater.inflate(R.layout.fragment_home, container, false)
    }

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)

        mInsertData.setOnClickListener {
            fragmentManager!!.beginTransaction()
                .replace(R.id.mFrameContainer, AddUserFragment())
                .addToBackStack(null)
                .commit()
        }

        mGetData.setOnClickListener {
            fragmentManager!!.beginTransaction()
                .replace(R.id.mFrameContainer, ReadUserFragment())
                .addToBackStack(null)
                .commit()
        }
        mDeleteData.setOnClickListener {
            fragmentManager!!.beginTransaction()
                .replace(R.id.mFrameContainer, DeleteUserFragment())
                .addToBackStack(null)
                .commit()
        }
        mUpdateData.setOnClickListener {
            fragmentManager!!.beginTransaction()
                .replace(R.id.mFrameContainer, UpdateFragment())
                .addToBackStack(null)
                .commit()
        }
        mGetDataList.setOnClickListener {
            fragmentManager!!.beginTransaction()
                .replace(R.id.mFrameContainer, GetDataIntoRecycler())
                .addToBackStack(null)
                .commit()
        }
    }


}
